mix.exs

```diff
@@ -41,7 +41,8 @@ defmodule Membrane.Opus.Plugin.Mixfile do
       {:membrane_common_c, "~> 0.3.0"},
       {:unifex, "~> 0.2.6"},
       {:dialyxir, "~> 1.0.0-rc.7", only: :dev, runtime: false},
-      {:ex_doc, "~> 0.21", only: :dev}
+      {:ex_doc, "~> 0.21", only: :dev},
+      {:membrane_element_file, "~> 0.3.0", only: [:dev, :test]}
```

Comment:
`only: :test` should be enough. You can do `MIX_ENV=test iex -S mix` when needed